### PR TITLE
Internationalization ( for now EN,DE have the default configs )

### PR DESCRIPTION
**Pull request by sereneinserenade**
- give users the option to write the tooltips themselves if they want
- give users the option to just define a language to set the default tooltips(if we have that language in the default config, otherwise use English)
- updated the email in the PyPi config file since now we have one
- making the development instructions commands correct in the README.md

### CODE REVIEW

File: django_tiptap/widgets.py [hunk 1]
```diff
@@ -29,4 +28,32 @@ def set_config(config: dict = None) -> dict:
...
+            langs: list = ["EN", "DE"]
+            givenLang: str = context["widget"]["config"].get("lang")
+
+            if (givenLang.upper() in langs):
+                context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()[givenLang.upper()]
+            else:
+                context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()["EN"]
+        else: 
+            context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()["EN"]
+
```
Comment (solen0id) [inline django_tiptap/widgets.py:58]: What do you think about moving this into a function in `configs.py` in order to keep concerns seperated & nice and clean? :)

File: django_tiptap/widgets.py [hunk 2]
```diff
@@ -29,4 +28,32 @@ def set_config(config: dict = None) -> dict:
     def get_context(self, *args, **kwargs) -> Dict[str, Any]:
         context = super().get_context(*args, **kwargs)
         context["widget"]["config"] = self.config
+
+        if (context["widget"]["config"].get("tooltips")):
```
Comment (solen0id) [inline django_tiptap/widgets.py:32]: In contrast to JS, it is more idiomatic in Python to not wrap the statement your evaluating in parentheses.  
Also, if you just want to check whether or not a given key is contained in a dict, without the need of assigning the corresponding value to a variable, you can use the following construct: `if key in dict:`

Putting it both together could look something like this:
```suggestion
        if "tooltips" in context["widget"]["config"]:
```

**Review by solen0id — COMMENTED** [review]
Cool stuff :rocket: 
Maybe we can refactor some of the config parsing and move it to another place, otherwise its looking good to me :)

File: django_tiptap/widgets.py (hunk 2 above)
Comment (sereneinserenade) [reply django_tiptap/widgets.py]: Cool Stuff ⚡

File: django_tiptap/widgets.py (hunk 1 above)
Comment (sereneinserenade) [reply django_tiptap/widgets.py]: yeah, that sounds cool, I'll move it there

Comment (sereneinserenade) [reply django_tiptap/widgets.py]: resolved

**Pull request merged**